src/libraries/System.Security.Cryptography.Cose/ref/System.Security.Cryptography.Cose.cs

```diff
@@ -0,0 +1,78 @@
+        public partial struct Enumerator : System.Collections.Generic.IEnumerator<(System.Security.Cryptography.Cose.CoseHeaderLabel Label, System.ReadOnlyMemory<byte> EncodedValue)>, System.Collections.IEnumerator, System.IDisposable
+        {
+            public readonly (System.Security.Cryptography.Cose.CoseHeaderLabel Label, System.ReadOnlyMemory<byte> EncodedValue) Current { get { throw null; } }
+            object System.Collections.IEnumerator.Current { get { throw null; } }
+            public void Dispose() { }
+            public bool MoveNext() { throw null; }
+            public void Reset() { }
+        }
+    }
+    public abstract partial class CoseMessage
+    {
+        internal CoseMessage() { }
+        public System.ReadOnlyMemory<byte>? Content { get { throw null; } }
+        public System.Security.Cryptography.Cose.CoseHeaderMap ProtectedHeaders { get { throw null; } }
+        public System.Security.Cryptography.Cose.CoseHeaderMap UnprotectedHeaders { get { throw null; } }
+        public static System.Security.Cryptography.Cose.CoseSign1Message DecodeSign1(byte[] cborPayload) { throw null; }
+    }
+    public sealed partial class CoseSign1Message : System.Security.Cryptography.Cose.CoseMessage
+    {
+        internal CoseSign1Message() { }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public static byte[] Sign(byte[] content, System.Security.Cryptography.Cose.CoseHeaderMap protectedHeaders, System.Security.Cryptography.Cose.CoseHeaderMap unprotectedHeaders, System.Security.Cryptography.AsymmetricAlgorithm key, System.Security.Cryptography.HashAlgorithmName hashAlgorithm, bool isDetached = false) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public static byte[] Sign(byte[] content, System.Security.Cryptography.ECDsa key, System.Security.Cryptography.HashAlgorithmName hashAlgorithm, bool isDetached = false) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public static byte[] Sign(byte[] content, System.Security.Cryptography.RSA key, System.Security.Cryptography.HashAlgorithmName hashAlgorithm, bool isDetached = false) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public bool Verify(System.Security.Cryptography.ECDsa key) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public bool Verify(System.Security.Cryptography.ECDsa key, System.ReadOnlySpan<byte> content) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
+        public bool Verify(System.Security.Cryptography.RSA key) { throw null; }
+        [System.Runtime.Versioning.UnsupportedOSPlatformAttribute("browser")]
```

Comment:
NIT: probably better to apply the attribute at the type level as all membrs have it except internal constructor

Reply:
Given that is a nit and the CI finished mostly clean, I will address this in a future PR.

Reply:
One a second thought, we shouldn't move the attribute to the type since the internal .ctor is used in an API (`CoseMessage.DecodeSign1`) that can be called from browser.